backend/index.js

```diff
@@ -0,0 +1,153 @@
+import { initializeApp } from 'firebase/app';
+import { getDatabase, ref, set, onValue, push, remove} from 'firebase/database';  
+
+
+const firebaseConfig = {
+    apiKey: "AIzaSyBSCJjXkCl7zH26AOle7xSO_lrJmMyl1hw",
+    authDomain: "earth-day-9d912.firebaseapp.com",
+    databaseURL: "https://earth-day-9d912-default-rtdb.firebaseio.com",
+    projectId: "earth-day-9d912",
+    storageBucket: "earth-day-9d912.appspot.com",
+    messagingSenderId: "296094286428",
+    appId: "1:296094286428:web:e425b2fafaef3b828c5c99",
+    measurementId: "G-633FC55T5Y"
+
+};

```

Comment:
I will move this to a separate js module, so we can use another database easily for testing, and to prevent publicly releasing credentials.
The config values will be in `backend/firebaseConfig.js`

Reply:
ok